tasks.yaml

```diff
@@ -89,7 +90,12 @@ tasks:
         with:
           assessment_results: /tmp/oscal-assessment-results.yaml
 
-  - name: test-upgrade
+  - name: test-latest-tag-flavor
```

Comment:
we shouldn't call tasks the same name across files since that can make it more confusing to know what is called where.

Reply:
(also pretty sure this is dead code)